src/main/java/com/quantumsoft/qupathcloud/dao/spec/SeriesPathBuilder.java

```diff
@@ -15,37 +15,55 @@
 
 package com.quantumsoft.qupathcloud.dao.spec;
 
+import static com.quantumsoft.qupathcloud.dao.Constants.DATASETS;
+import static com.quantumsoft.qupathcloud.dao.Constants.DICOM_STORES;
+import static com.quantumsoft.qupathcloud.dao.Constants.DICOM_WEB;
+import static com.quantumsoft.qupathcloud.dao.Constants.LOCATIONS;
+import static com.quantumsoft.qupathcloud.dao.Constants.PROJECTS;
+import static com.quantumsoft.qupathcloud.dao.Constants.SERIES;
+import static com.quantumsoft.qupathcloud.dao.Constants.STAGE;
+import static com.quantumsoft.qupathcloud.dao.Constants.STUDIES;
+
 import com.quantumsoft.qupathcloud.exception.QuPathCloudException;
 
-import static com.quantumsoft.qupathcloud.dao.Constants.*;
+/**
+ * SeriesPathBuilder to build a path for access to Series in chosen Project, Location, Dataset,
+ * DICOM Store and Study. If Study is null then will be built a path for all Series in chosen
+ * Project, Location, Dataset, and DICOM Store.
+ */
```

Comment:
SeriesPathBuilder builds a path to access the Series in the specified Project, Location, Dataset, DICOM Store and Study. If Study is null, then the path will be built for all Series in the specified Project, Location, Dataset, and DICOM Store.